app/assets/sass/_wmt/_wmt-data-table.scss

```diff
@@ -61,6 +61,18 @@ table.data-table tfoot tr {
     background: rgb(199, 192, 192);
 }
 
+table.data-table tfoot tr th {
+    background: rgb(199, 192, 192);
+    font-size: 16px;
+    font-weight: normal;
+}
+
+table.data-table tfoot tr th:first-child {
```

Comment:
This is to style the left-most column of the summary row to match the current style